### DISCUSSION TITLE
Too Much virtual memory for executing node program

### DISCUSSION THREAD

**Discussion by ampandey-AMD** [neoclide/coc.nvim#4454] [General]
why does coc.nvim build/index.js takes too much virtual memory?
![image](https://user-images.githubusercontent.com/86796896/209350144-b8aa05cc-bc4e-4b78-beb4-a9f529dbc5cc.png)

**Comment by chemzqm** [comment]
It's expected that NodeJS program did takes much more memory.
You can inspect by Chrome `:h coc-api-debug`.
The process may not terminated as expected on your system.

**Comment by jackielii** [comment]
I'm not sure this is an actual problem. Virtual memory is not related to actual memory usage. The "RES" column which is actual memory usage says "115M" usage which is pretty normal. On my machine some big projects take us >500MB easily.

I think you should only worry when it gets slow, it's a tool, getting the job done is the important bit.
